sdk/storage/Azure.Storage.DataMovement.Blobs/src/Azure.Storage.DataMovement.Blobs.csproj

```diff
@@ -6,33 +6,25 @@
     <AssemblyTitle>Microsoft Azure.Storage.DataMovement.Blobs client library</AssemblyTitle>
     <Version>12.0.0-beta.7</Version>
     <DefineConstants>BlobDataMovementSDK;$(DefineConstants)</DefineConstants>
-    <PackageTags>Microsoft Azure Storage DataMovement, DataMovement, Microsoft, Azure, StorageScalable, azureofficial</PackageTags>
+    <PackageTags>Microsoft Azure Storage DataMovement Blobs AzureStorage BlobStorage azureofficial</PackageTags>
```

Comment:
Tags are space or comma delimited so we had duplicate tags. Space delimited is the recommended option according to Visual Studio.